python/lsst/ap/verify/measurements/profiling.py

```diff
@@ -27,44 +27,44 @@
 
 from __future__ import absolute_import, division, print_function
 
-__all__ = ["measure_runtime"]
+__all__ = ["measureRuntime"]
 
 import astropy.units as u
 
 import lsst.verify
 
 
-def measure_runtime(metadata, task_name, metric_name):
+def measureRuntime(metadata, taskName, metricName):
     """Computes a wall-clock measurement from metadata provided
     by @pipe.base.timeMethod.
 
     Parameters
     ----------
     metadata: `lsst.daf.base.PropertySet`
         The metadata to search for timing information.
-    task_name: `str`
+    taskName: `str`
         The name of the Task, e.g., "processCcd". SubTask names must be the
         ones assigned by the parent Task and may be disambiguated using the
         parent Task name, as in "processCcd:calibrate".
-        If `task_name` matches multiple runs of a subTask in different
+        If `taskName` matches multiple runs of a subTask in different
         contexts, the information for only one run will be provided.
-    metric_name: `str`
+    metricName: `str`
         The fully qualified name of the metric being measured, e.g.,
         "pipe_tasks.ProcessCcdTime"
 
     Returns
     -------
-    an `lsst.verify.Measurement` for `metric_name`, or `None` if the timing
-    information for `task_name` is not present in `metadata`
+    an `lsst.verify.Measurement` for `metricName`, or `None` if the timing
```

Comment:
Returns is formatted like Parameters, even if you need to make up the name of the return variable. For example, I'd write this one as:

```rst
Returns
-------
measurement : `lsst.verify.Measurement`
    The metric measurement, or `None` if timing information for ``taskName`` is not present in the ``metadata``.
```